docs/data/joy/getting-started/templates/email/App.tsx

```diff
@@ -108,12 +106,7 @@ export default function EmailExample() {
               justifyContent: 'space-between',
             }}
           >
-            <Box
-              sx={{
-                alignItems: 'center',
-                gap: 1,
-              }}
-            >
+            <Box sx={{ alignItems: 'center', gap: 1 }}>
```

Comment:
Try to keep the line count as low as possible. This is often a concern relative to Tailwind CSS: how the screen is not tall enough to see the structure of the component.